cmd/ttn-lw-cli/commands/simulate.go

```diff
@@ -637,13 +650,43 @@ var (
 			)
 		},
 	}
+	simulateApplicationUplinkCommand = &cobra.Command{
+		Use:   "application-uplink [application-id] [device-id]",
+		Short: "Simulate application uplink message from an end device",
```

Comment:
I think the description should be more clear about the fact that this "simulates an application-layer uplink message from an end device, sent directly to the application server"